### ISSUE TITLE
cc-shim: 3.0.0 still is shown as alpha

### ISSUE THREAD

**Issue by gorozco1** [clearcontainers/shim#74]
Because of this line:
https://github.com/clearcontainers/shim/blob/master/configure.ac#L16

`cc-runtime cc-env` shows:
```
[Shim]
  Type = "ccShim"
  Version = "shim version: 3.0.0-alpha.4 (commit: 739e5e2)"
  [Shim.Location]
    Path = "/usr/libexec/clear-containers/cc-shim"
    Resolved = "/usr/libexec/clear-containers/cc-shim"
```

**Comment by gorozco1** [comment]
This needs to be fixed in the next release of cc-shim

**Comment by gorozco1** [comment]
Fixed on release 3.0.1 closing
